index.html

```diff
@@ -19,13 +19,13 @@
   <div class="HEAD">
     <header class="header-container">
       <div class="Logo">
```

Comment:
[ OPTIONAL ] Please make sure there is no horizontal scroll. It is because some content is overflowing from the container.